src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -0,0 +1,433 @@
+            $wrappedTraits[$key] = new ReflectionClass($trait);
+        }
+        return $wrappedTraits;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getTraitNames()
+    {
+        return $this->betterReflectionClass->getTraitNames();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getTraitAliases()
+    {
+        return $this->betterReflectionClass->getTraitAliases();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isTrait()
+    {
+        return $this->betterReflectionClass->isTrait();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isAbstract()
+    {
+        return $this->betterReflectionClass->isAbstract();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isFinal()
+    {
+        return $this->betterReflectionClass->isFinal();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getModifiers()
+    {
+        return $this->betterReflectionClass->getModifiers();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isInstance($object)
+    {
+        return $this->betterReflectionClass->isInstance($object);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function newInstance($args = null, $_ = null)
+    {
+        throw new Exception\NotImplemented('Not implemented');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function newInstanceWithoutConstructor()
+    {
+        throw new Exception\NotImplemented('Not implemented');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function newInstanceArgs(array $args = null)
+    {
+        throw new Exception\NotImplemented('Not implemented');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getParentClass()
+    {
+        return new ReflectionClass($this->betterReflectionClass->getParentClass());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function isSubclassOf($class)
+    {
+        return $this->betterReflectionClass->isSubclassOf($class);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getStaticProperties()
+    {
+        throw new Exception\NotImplemented('Not implemented');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getStaticPropertyValue($name, $default = null)
+    {
+        throw new Exception\NotImplemented('Not implemented');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setStaticPropertyValue($name, $value)
+    {
+        throw new Exception\NotImplemented('Not implemented');
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDefaultProperties()
+    {
+        $properties = $this->betterReflectionClass->getDefaultProperties();
+
+        $wrappedProperties = [];
+        foreach ($properties as $key => $property) {
+            $wrappedProperties[$key] = new ReflectionProperty($property);
+        }
+        return $wrappedProperties;
```

Comment:
IIRC, `getDefaultProperties()` returns the actual values. This code is currently causing `var_export does not handle circular references` in ProxyManager sources.